Resources/Prototypes/_Mono/Entities/Structures/walls.yml

```diff
@@ -146,7 +146,7 @@
         - FullTileMask
         layer:
         - WallLayer
-        density: 9000 # No ram-meta
+        density: 1000 # No ram-meta
```

Comment:
that's normal wall density
you seem to have weird density changes on other walls too
i think density is okay as it is